### PR TITLE
Add hierarchy for data tiering

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

This moves the data tiering section into a directory, and splits the content into an index and `move-data`.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/web-documentation/issues/173

### CODE REVIEW

File: timescaledb/how-to-guides/page-index/page-index.js [hunk 1]
```diff
@@ -346,8 +346,14 @@ module.exports = [
...
       {
+        title: "Data Tiering",
         href: "data-tiering"
-      },
+        children: [
+          {
+            href: "move-data"
+          }
+        ]
+      }
```
Comment (Loquacity) [inline timescaledb/how-to-guides/page-index/page-index.js:356]:
```suggestion
      },
```

**Comment by Loquacity** [comment]
image

**Comment by mfreed** [comment]
Is the badging working?  Looks like something is off with the current docs - or is that a different story?

image

**Review by mfreed — APPROVED** [review]

**Comment by Loquacity** [comment]
> Is the badging working? Looks like something is off with the current docs - or is that a different story?
> 
> image

Yeah, I'm not sure. Might be a @helrabelo or @tedsczelecki question? I guess it's going to become obsolete with the new tagging that we're setting up, though.

File: timescaledb/how-to-guides/data-tiering/move-data.md [hunk 2]
```diff
@@ -52,7 +20,6 @@ Here we are creating a tablespace called `history` that will be
 owned by the default `postgres` user, using the storage mounted at `/mnt/history`.
 
 ## Move Chunks :community_function: [](move_chunks)
```
Comment (Loquacity) [inline timescaledb/how-to-guides/data-tiering/move-data.md:22]:
```suggestion
## Move Chunks [](move_chunks)
```

**Pull request merged**